File: testsuite/results/bug78.nim
proc loop*(): cint =
  var n: cint = 0
  var i: cint = 2
  while dec(i) > 0:
    if i == 1:
      continue
    inc(n, i)
  n = 0
  i = 2
  while i > 0:
    if i == 1:
      dec(i)
      continue
    inc(n, i)
    dec(i)
  return n

File: testsuite/tests/bug78.c
int loop() {
    int n = 0;
    int i = 2;
    while (i-- > 0) {
        if (i == 1) continue;
        n += i;
    }

    for (n=0, i=2; i>0; i--) {
        if (i == 1) continue;
        n += i;
    }
    return n;
}
